modules/terraform-cdp-deploy/modules/aws/variables.tf

```diff
@@ -152,11 +147,11 @@ variable "public_subnet_ids" {
   description = "List of public subnet ids."
 }
 
-variable "private_subnet_ids" {
-  type        = list(string)
-  description = "List of private subnet ids."
+# variable "private_subnet_ids" {
```

Comment:
@jimright Can you please explain this part? Why is removing private_subnet_ids being removed?

Reply:
This is removed because the linter flagged that it is not being used in the aws deploy **sub-module**. (Note that the terraform-cdp-deploy module still requires the private_subnets as an input.)

The sub-module instead uses the [subnets_for_cdp](https://github.com/cloudera-labs/terraform-cdp-modules/blob/e6907800000bcaa33c11a1730700fbbed8d42f86/modules/terraform-cdp-deploy/modules/aws/variables.tf#L155-L160) variable when creating the CDP environment. This already has the private and potentially the public subnets depending on the deployment template.
The public subnets input variable is still required for endpoint_access_gateway_subnet_ids.

I have commented rather than removed as I feel we might need re-introduce this again as part of some advanced networking work.